### PR TITLE
Add config for testing against a FiaB

### PR DESCRIPTION
**Pull request by breilly2**
I plan to also add some short docs about starting a FiaB for this. What would be most appropriate, a `.md` file or something in the wiki or something else?

### CODE REVIEW

**Comment by zarsky-broad** [comment]
I'd say something in the wiki.

File: config/fiab.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "agoraUrlRoot": "https://agora-fiab.dsde-dev.broadinstitute.org:20443",
+  "calhounUrlRoot": "https://terra-calhoun-dev.appspot.com",
```
Comment (panentheos) [inline config/fiab.json:3]: Note: calhoun doesn't run inside a fiab, and dev points to dev Sam for auth purposes, so that wouldn't work as expected.

**Comment by breilly2** [comment]
Added some newer configs and wrote a wiki page: https://github.com/DataBiosphere/saturn-ui/wiki/Develop-against-FiaB

**Review by panentheos — APPROVED** [review]
Looks good.

**Pull request merged**